docs/app/pages/components/components-sections/buttons/split-button-dropdowns/snippets/app.microfocus.html

```diff
@@ -1,68 +1,151 @@
-<div class="btn-group" dropdown #dropdown="bs-dropdown" [autoClose]="true">
-    <!-- Standalone Action Buttons -->
-    <button type="button" class="btn button-secondary">Action</button>
+<div class="row">
+    <div class="col-md-6">
+        <h4>Raised</h4>
+        <div class="btn-group">
+            <button type="button" class="btn button-primary">
+                Actions
+            </button>
+            <button
+                type="button"
+                class="btn button-primary dropdown-toggle"
+                [uxMenuTriggerFor]="menu">
```

Comment:
You've remove a whole bunch of attributes from the original example here. I suggest reverting these changes and starting over from the existing code, taking care not to remove anything.